packages/client/src/utils/LoggingStaticJsonRpcProvider.ts

```diff
@@ -0,0 +1,34 @@
+import { StaticJsonRpcProvider } from '@ethersproject/providers'
+import { Logger, randomString } from '@streamr/utils'
+
+export class LoggingStaticJsonRpcProvider extends StaticJsonRpcProvider {
+    private readonly logger = new Logger(module)
+
+    override async send(method: string, params: any[]): Promise<any> {
+        const traceId = randomString(5)
+        const startTime = Date.now()
+        const logContext = {
+            traceId,
+            method,
+            params,
+            connection: this.connection
+        }
+        this.logger.debug('Sending request', logContext)
```

Comment:
```suggestion
        this.logger.debug('Send request', logContext)
```

https://github.com/streamr-dev/network/blob/main/developer-docs/logging-practices.md